Make "Learning the API" less off-putting

TehShrike opened the pull request:
The line "Documentation on both is scarce at the time of this writing, but you can read existing code and learn from it" screams "bugger off, you're not supposed to be making a plugin".

Considering that there appears to be decent API documentation, it seems much more encouraging to just link to it.

Beyond that, "you will first have to extract its content to a folder if you want to take a look at the code within. As an exercise, you can try creating a build system to do that on demand" says "learning is hard, and you should do work before you can start".

This pull request attempts to make the final lines of this introduction to plugins less off-putting.

TehShrike committed:
Make "Learning the API" less off-putting

keith-hall commented on source/extensibility/plugins.rst:
```diff
@@ -216,12 +216,11 @@ for plugins go, this a very bad one.
-its content to a folder if you want to take a look at the code within. As an
-exercise, you can try creating a build system to do that on demand, and a
-project file to be able to peek at the sample code easily.
+its contents to a folder if you want to take a look at the code within.
```
we could probably recommend https://packagecontrol.io/packages/PackageResourceViewer here :)

TehShrike committed:
Adding link to PackageResourceViewer

Suggested by @keith-hall in https://github.com/guillermooo/sublime-undocs/pull/240/files#r123324494

FichteFoll commented:
That entire page needs a rework, but little patches won't hurt until someone gets to it. Thanks.
